request.go

```diff
@@ -656,6 +663,32 @@ func handleStringSlice(attribute interface{}) (reflect.Value, error) {
 	return reflect.ValueOf(values), nil
 }
 
+func handleNullable(
+	attribute interface{},
+	args []string,
+	structField reflect.StructField,
+	fieldValue reflect.Value) (reflect.Value, error) {
+
+	if a, ok := attribute.(string); ok {
+		if bytes.Equal([]byte(a), []byte("null")) {
+			return reflect.ValueOf(nil), nil
+		}
+	}
+
+	var rgx = regexp.MustCompile(`\[(.*)\]`)
+	rs := rgx.FindStringSubmatch(fieldValue.Type().Name())
+
+	attrVal, err := unmarshalAttribute(attribute, args, structField, SupportedNullableTypes[rs[1]])
+	if err != nil {
+		return reflect.ValueOf(nil), err
+	}
+
+	fieldValue.Set(reflect.MakeMap(fieldValue.Type()))
+	fieldValue.SetMapIndex(reflect.ValueOf(true), attrVal)
+
+	return fieldValue, nil
```

Comment:
I think you can drop the regex and SupportedNullableTypes map by getting the zero value of the Elem() type

```suggestion
	innerType := fieldValue.Type().Elem()
	zeroValue := reflect.Zero(innerType)

	attrVal, err := unmarshalAttribute(attribute, args, structField, zeroValue)
	if err != nil {
		return reflect.ValueOf(nil), err
	}

	fieldValue.Set(reflect.MakeMap(fieldValue.Type()))
	fieldValue.SetMapIndex(reflect.ValueOf(true), attrVal)

	return fieldValue, nil
```

Reply:
Fantastic. Thank you!